services/distribution/src/main/java/app/coronawarn/server/services/distribution/objectstore/publish/LocalFile.java

```diff
@@ -73,10 +73,10 @@ protected String createS3Key(Path file, Path rootFolder) {
    * Value for the <code>content-type</code> header.
    *
    * @return Either <a href="https://www.iana.org/assignments/media-types/application/zip">zip</a> or
-   *         <a href="https://www.iana.org/assignments/media-types/application/json">json</a>.
+   *     <a href="https://www.iana.org/assignments/media-types/application/json">json</a>.
    */
   public String getContentType() {
-    if (isConfigFile() || isStatisticFile() || isKeyFile()) {
+    if (isConfigFile() || isStatisticFile() || isKeyFile() || isQrPosterTemplate()) {
```

Comment:
What the ticket really wanted, was to ensure the content type for QR poster templates. Although is done via the `isQrPosterTemplate` method, is not _explicitly_ and directly tested. Maybe add a test for that?

Reply:
I have just added a new test